shared/editor/embeds/Tldraw.tsx

```diff
@@ -7,13 +7,13 @@ function Tldraw(props: Props) {
     <Frame
       {...props}
       src={props.attrs.href}
-      title="Tldraw.com Embed"
+      title="Tldraw.com (beta) Embed"
       referrerPolicy="origin"
       border
     />
   );
 }
 
-Tldraw.ENABLED = [new RegExp("https?://www\\.tldraw\\.com/r/(.*)$")];
+Tldraw.ENABLED = [new RegExp("https?://beta.tldraw.com/r/(.*)")];
```

Comment:
Oops! Should escape the `.`(dot) in regex here using `\\` as before otherwise it allows arbitrary domains like `https://betatldraw.com` which might be a trap. Refer https://github.com/outline/outline/issues/3899 for details.

Also don't forget to update tests :)

Reply:
Oh yes sorry ! I will update accordingly

Reply:
Done and done